warp10/src/main/java/io/warp10/script/functions/TOPICKLE.java

```diff
@@ -16,29 +16,45 @@
 
 package io.warp10.script.functions;
 
+import io.warp10.continuum.gts.GTSEncoder;
+import io.warp10.continuum.gts.GTSPickler;
+import io.warp10.continuum.gts.GeoTimeSerie;
 import io.warp10.script.NamedWarpScriptFunction;
 import io.warp10.script.WarpScriptException;
 import io.warp10.script.WarpScriptStack;
 import io.warp10.script.WarpScriptStackFunction;
 
 import java.io.IOException;
 
+import net.razorvine.pickle.Pickler;
 import net.razorvine.pyro.serializer.PickleSerializer;
 
 /**
  * Encode an object as Pickle
  */
 public class TOPICKLE extends NamedWarpScriptFunction implements WarpScriptStackFunction {
+
+  private static final TOGTS TOGTS = new TOGTS("TOGTS");
```

Comment:
TOGTS is not the name under which this function is usually used, please use the name ->GTS by making it a constant in WarpScriptLib.